platform-hub-api/spec/support/user_activation_helpers.rb

```diff
@@ -0,0 +1,53 @@
+module UserActivationHelpers
+
+  RSpec.shared_examples "verify responses to exceptions" do |action|
```

Comment:
The `verify responses to exceptions` bit might be a bit too generic? Could say something like `verify responses to exceptions for user activation`?